src/hotspot/share/classfile/classFileError.cpp

```diff
@@ -78,6 +78,25 @@ void ClassFileParser::classfile_parse_error(const char* msg,
                      msg, name, signature, _class_name->as_C_string());
 }
 
+void ClassFileParser::classfile_icce_error(const char* msg,
```

Comment:
Is there a reason we can't just use THROW_MSG at the callsites rather than introducing these helper functions?

Reply:
I think it's intended to match existing usage.